packages/storefront-lwc/src/modules/commerce/basket/basket.js

```diff
@@ -40,15 +41,12 @@ export default class Basket extends LightningElement {
         return ShoppingBasket.basket.selectedShippingMethodId;
     }
```

Comment:
Interesting: selectedShippingMethodId === hasProducts? I would think `this.products.length`

Reply:
This line of code does not exist in the file.